Automattic/jetpack
# Blocks: update checkout links to include the full Jetpack plan.

#### Changes proposed in this Pull Request:

Calypso now seems to accept the full, jetpack_ prefixed plan slugs in its checkout page.
When provided with a non-prefixed slug, it tries to add a WordPress.com Plan to the cart and that is not possible on Jetpack sites.


#### Jetpack product discussion

* Internal reference: p1607628394297200-slack-CBG1CP4EN

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

You'll want to test on both Jetpack and WordPress.com.

- In Jetpack, start from a connected site with the nudges activated via `add_filter( 'jetpack_block_editor_enable_upgrade_nudge', '__return_true' );`
- On WordPress.com, start from a site with a free plan with D54135-code.

* Go to Posts > Add New and add a Pay with PayPal block, as well as a Payments block.
* The upgrade nudge should appear above each block.
* When clicking on it, it should bring you to Calypso, with the right plan in the cart.

#### Proposed changelog entry for your changes:

* N/A
